docs/reference/export-metrics.mdx

```diff
@@ -0,0 +1,166 @@
+---
+title: Exporting Metrics (Prometheus)
+sidebar_position: 1200
+description: Using OpenTelemetry to export gadget metrics to Prometheus
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+Inspektor Gadget supports exporting metrics to Prometheus using the [metrics
+exporter](https://opentelemetry.io/docs/specs/otel/metrics/sdk_exporters/prometheus/).
+In order to do so, you need to [enable the metrics
+listener](#enabling-the-metrics-listener) and [configure each gadget to actively
+expose its metrics](#enabling-export-for-gadgets).
+
+## Enabling the metrics listener
+
+In order to enable the metrics listener, you need to set the
+`otel-metrics-listen` parameter to `true`:
+
+<Tabs groupId="env">
+    <TabItem value="kubectl-gadget" label="kubectl gadget">
+        ```bash
+        WIP: Headless mode for kubectl gadget is under development
+        ```
+    </TabItem>
+
+    <TabItem value="ig" label="ig">
+        ```bash
+        ig daemon --otel-metrics-listen=true
+        ```
+    </TabItem>
+</Tabs>
+
+## Enabling export for gadgets
+
+Some gadgets have the ability to export metrics. To know if a gadget supports
+exporting metrics, you can check the *Exporting metrics* section of the gadget's
+documentation. Such section will tell you whether or not the gadget exports
+metrics and, in case, what metrics it exports. The
+[profile_blockio](../gadgets/profile_blockio#exporting-metrics) gadget is an
+example of a gadget that exports metrics.
```

Comment:
Make it more concise? 

```suggestion
Some gadgets have the ability to export metrics. The *Exporting metrics* section
of the gadget's documentation will tell you whether or not the gadget exports
metrics and, in case, what metrics it exports. The
[profile_blockio](../gadgets/profile_blockio#exporting-metrics) gadget is an
example of a gadget that exports metrics.
```